react/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -1,10 +1,36 @@
 import { isArray } from './utils';
 
-export function min() {
-  isArray();
-  return Math.min();
+export function min(...numbers) {
+  if(arguments.length === 0){
+    return undefined;
+  }
+
+  if(isArray(...numbers)){
+    return Math.min.apply(Math, ...numbers);
+  }
+  else {
+    return Math.min.apply(Math, numbers);
+  }
+
 }
 
-export function copy() {
+export function copy(element) {
+  if(isArray(element)){
+    return [...element];
+  }
+  else {
+    return {...element};
```

Comment:
If `element` is not an object, this will explode! Assume this function receives an array or an object and throw an exception if it's not the case.